### PR TITLE
Update color tokens

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #57 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

* Update all dependencies, most importantly the design tokens
* Refactor color tokens by isolating the color tokens and creating tier 3 tokens for repeatable rules

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

## Verification

- [ ] Tier 3 tokens are appropriately named
- [ ] Tier 3 tokens are pointing to correct tier 2 tokens

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: demo/api.md [hunk 1]
```diff
@@ -539,4 +539,23 @@ The following example illustrates a `<auro-banner>` custom element with the `rou
...
+### Theme Support
+
+The component may be restyled using the following code sample and changing the values of the following token(s).
+
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../src/tokens.scss) -->
+<!-- The below code snippet is automatically added from ./../../src/tokens.scss -->
+
+```scss
+:host {
+  --ds-auro-banner-text-color: var(--ds-color-text-primary-default, $ds-color-text-primary-default);
```
Comment (jason-capsule42) [inline demo/api.md:553]: Alphabetize

File: demo/api.md [hunk 2]
```diff
@@ -539,4 +539,23 @@ The following example illustrates a `<auro-banner>` custom element with the `rou
...
+  --ds-auro-banner-disclaimer-text-color: var(--ds-color-text-secondary-default, $ds-color-text-secondary-default);
+  --ds-auro-banner-title-text-color: var(--ds-color-text-emphasis-default, $ds-color-text-emphasis-default);
+  --ds-auro-banner-icon-background-color: var(--ds-color-icon-tertiary-default, $ds-color-icon-tertiary-default);
```
Comment (jason-capsule42) [inline demo/api.md:556]:
```suggestion
  --ds-auro-banner-icon-container-color: var(--ds-color-icon-tertiary-default, $ds-color-icon-tertiary-default);
```

Why are we using a icon token for a background rule?

File: demo/api.md [hunk 3]
```diff
@@ -539,4 +539,23 @@ The following example illustrates a `<auro-banner>` custom element with the `rou
...
+  --ds-auro-banner-wrapper-background-color: var(--ds-color-brand-midnight-400, $ds-color-brand-midnight-400);
```
Comment (jason-capsule42) [inline demo/api.md:557]: We have to figure out how to extract the Tier 1 brand color out of here so that it works with the Hawaiian theme, etc.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/tokens.scss [hunk 4]
```diff
@@ -0,0 +1,8 @@
+:host {
+  --ds-auro-banner-text-color: var(--ds-color-text-primary-default, $ds-color-text-primary-default);
+  --ds-auro-banner-disclaimer-text-color: var(--ds-color-text-secondary-default, $ds-color-text-secondary-default);
+  --ds-auro-banner-title-text-color: var(--ds-color-text-emphasis-default, $ds-color-text-emphasis-default);
+  --ds-auro-banner-icon-background-color: var(--ds-color-icon-tertiary-default, $ds-color-icon-tertiary-default);
+  --ds-auro-banner-wrapper-background-color: var(--ds-color-brand-midnight-400, $ds-color-brand-midnight-400);
+  --ds-auro-banner-wrapper-border-color: var(--ds-color-border-primary-default, $ds-color-border-primary-default);
```
Comment (blackfalcon) [inline src/tokens.scss:7]:
```suggestion
// Import Auro tokens
@import "./../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables";

:host {
  --ds-auro-banner-text-color: var(--ds-color-text-primary-default, #{$ds-color-text-primary-default});
  --ds-auro-banner-disclaimer-text-color: var(--ds-color-text-secondary-default, #{$ds-color-text-secondary-default});
  --ds-auro-banner-title-text-color: var(--ds-color-text-emphasis-default, #{$ds-color-text-emphasis-default});
  --ds-auro-banner-icon-background-color: var(--ds-color-icon-tertiary-default, #{$ds-color-icon-tertiary-default});
  --ds-auro-banner-wrapper-background-color: var(--ds-color-brand-midnight-400, #{$ds-color-brand-midnight-400});
  --ds-auro-banner-wrapper-border-color: var(--ds-color-border-primary-default, #{$ds-color-border-primary-default});
```

Also, `$ds-color-icon-tertiary-default` is ** not** a token?

**Review by blackfalcon — CHANGES_REQUESTED** [review]

**Comment by blackfalcon** [comment]
I am seeing this in a few repos where there are tier 3 tokens being defined, when using this syntax pattern, Sass Interpolation is needed to process the Sass var into a value. 

```
--ds-auro-banner-wrapper-border-color: var(--ds-color-border-primary-default, #[$ds-color-border-primary-default});
```

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]
I made several updates based on the previous review comments in an attempt to get this done.

However, there is one issue that still needs attention. All the banner images are misaligned on the bottom of the banner. This needs to be fixed before this can release to beta.

**Comment by jordanjones243** [comment]
> I made several updates based on the previous review comments in an attempt to get this done.
> 
> However, there is one issue that still needs attention. All the banner images are misaligned on the bottom of the banner. This needs to be fixed before this can release to beta.

The issue that is seen has been verified to appear in the `main` branch as well, meaning the work from this repo can proceed.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**